pom.xml

```diff
@@ -23,12 +23,12 @@
   <parent>
     <groupId>org.hawkular</groupId>
     <artifactId>hawkular-parent</artifactId>
-    <version>43</version>
+    <version>46</version>
     <relativePath/>
   </parent>
   <groupId>org.hawkular.client</groupId>
   <artifactId>hawkular-java-client</artifactId>
-  <version>0.0.2-SNAPSHOT</version>
+  <version>0.0.16-SNAPSHOT</version>
```

Comment:
@jotak I updated the version to be the same as what hawkular-services version the client is targeting. Happy to revert, but thought it was a good idea if the versions matched to make it easier for the end user.